examples/medplum-patient-intake-demo/src/pages/QuestionnaireCustomizationPage.tsx

```diff
@@ -0,0 +1,59 @@
+import { normalizeErrorString } from '@medplum/core';
+import { Questionnaire } from '@medplum/fhirtypes';
+import { Document, QuestionnaireBuilder, useMedplum, useMedplumProfile, useResource } from '@medplum/react';
+import { useCallback } from 'react';
+import { showNotification } from '@mantine/notifications';
+import { Loading } from '../components/Loading';
+import { useNavigate, useParams } from 'react-router-dom';
+import { IconCircleCheck, IconCircleOff } from '@tabler/icons-react';
+
+export function QuestionnaireCustomizationPage(): JSX.Element {
+  const navigate = useNavigate();
+  const medplum = useMedplum();
+  const profile = useMedplumProfile();
+  const { questionnaireId } = useParams();
+
+  const questionnaire = useResource<Questionnaire>({ reference: `Questionnaire/${questionnaireId}` });
```

Comment:
@rahul1 Here we could either use the questionnaire in the context or fetch it again. I decided to fetch it again as it was more consistent with the URL structure and would provide the two approaches to the example. LMK if would prefer to always do it via context and I can change it.